tests/common.py

```diff
@@ -57,6 +64,7 @@ def skip_if_not_linux(func):
 TRACE_VERSION = 1
 
 
+@skipUnless(tracer, "Tracer is not installed and cached data is not present")
```

Comment:
this does not match the original behavior - this will skip if tracer is not installed even if cached data is present